cob_twist_controller/src/cob_twist_controller.cpp

```diff
@@ -37,8 +37,8 @@
 
 #include "cob_obstacle_distance/Registration.h"
 
-#define DEBUG_BASE_ACTIVE    0
-#define DEBUG_BASE_COMP     0
+
+//ToDo: Should we re-add DEBUG_BASE_COMPENSATION stuff?
```

Comment:
I would suggest a cob_twist_controller "global" DEBUG switch. So other modules could also use the define if desired. 

An additional advantage is that it's possible to have several build configurations in IDEs so it would be possible to build one with DEBUG on and one with DEBUG off.